lib/python/rose/apps/rose_bunch.py

```diff
@@ -342,11 +342,16 @@ def connect(self):
     def create_tables(self):
         """Create tables as appropriate"""
         existing = []
+        first_run = os.environ["CYLC_TASK_SUBMIT_NUMBER"] == "1"
```

Comment:
One problem with using the submit number is that it gets incremented for each *submit*, so if the first submit results in a submission failure, you will be in trouble here.